### PR TITLE
feat: get rid of cloudtest

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: Denis Tingaikin <denis.tingajkin@xored.com>

## Motivation

Since we decided to postpone adding a clusterctl (we want test default cni of public clusters) we need to provide a context for folks on how to setup gke clusters for nsm. For this we're deleting cloudtest tool and start use gcloud cli directly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,22 @@ GITHUB_RUN_NUMBER
...
 
 and then runs the [gke-start.sh](scripts/gke-start.sh)
+
+## Setup GKE cluster
+
+```bash
+gcloud container clusters create "${GKE_CLUSTER_NAME}" --project="${GKE_PROJECT_ID}" --machine-type="${GKE_CLUSTER_TYPE}" --num-nodes="${GKE_CLUSTER_NUM_NODES}" --zone="${GKE_CLUSTER_ZONE}" -q
+gcloud container clusters get-credentials "${GKE_CLUSTER_NAME}" --project="${GKE_PROJECT_ID}" --zone="${GKE_CLUSTER_ZONE}"
+gcloud compute firewall-rules create allow-proxy-nsm --action ALLOW --rules tcp:80 --project="${GKE_PROJECT_ID}"
+gcloud compute firewall-rules create allow-nsm --action ALLOW --rules tcp:5000-5100 --project="${GKE_PROJECT_ID}"
```
Comment (edwarnicke) [inline README.md:31]: Question... why is this a range of ports?

File: README.md [hunk 2]
```diff
@@ -21,3 +21,22 @@ GITHUB_RUN_NUMBER
 ```
 
 and then runs the [gke-start.sh](scripts/gke-start.sh)
+
+## Setup GKE cluster
+
+```bash
+gcloud container clusters create "${GKE_CLUSTER_NAME}" --project="${GKE_PROJECT_ID}" --machine-type="${GKE_CLUSTER_TYPE}" --num-nodes="${GKE_CLUSTER_NUM_NODES}" --zone="${GKE_CLUSTER_ZONE}" -q
+gcloud container clusters get-credentials "${GKE_CLUSTER_NAME}" --project="${GKE_PROJECT_ID}" --zone="${GKE_CLUSTER_ZONE}"
+gcloud compute firewall-rules create allow-proxy-nsm --action ALLOW --rules tcp:80 --project="${GKE_PROJECT_ID}"
```
Comment (edwarnicke) [inline README.md:30]: Do we really need to allow *all* port 80 traffic in?  Of could we do something more constrainted?

Comment (denis-tingaikin) [reply README.md]: I found that we don't need this for single cluster testing.

File: README.md (hunk 1 above)
Comment (denis-tingaikin) [reply README.md]: It was an artifact from the old monorepo. Fixed!

**Pull request merged**